typescript/db/src/resolvers/workspace.ts

```diff
@@ -78,10 +84,58 @@ const workspaces = async (
   );
 };
 
+type CreateTutorialDatasetProps = Context & {
+  createdWorkspaceId: string;
+};
+
+const createTutorialDatasetInWorkspace = async ({
+  repository,
+  req,
+  user,
+  createdWorkspaceId,
+}: CreateTutorialDatasetProps) => {
+  const now = new Date();
+  const createdWorkspace = await repository.workspace.get(
+    { id: createdWorkspaceId },
+    user
+  );
```

Comment:
The function is being called from the createWorkspace resolver. You should try to directly reuse the object returned by the creation instead of querying the DB.

Reply:
`workspace.add` does not return an object but only an ID